test/ManticoreBackupTest.php

```diff
@@ -257,7 +271,7 @@ class ManticoreMockedClient extends ManticoreClient {
   protected Closure $timeout_fn;
 
   /**
-   * Set delay timeout that we will use in case of calling freeze indexes
+   * Set delay timeout that we will use in case of calling freeze tables
```

Comment:
Pls remind me why we need the timeout.

Reply:
This is mocked class used only for testing purpose, it's required to do timeout to emulate test cases with interruption of the process in the middle. It's just workaround for tests only